### PR TITLE
[zh]Sync kubeadm-6

### PR DESCRIPTION
**Pull request by kinzhi**
related issue:https://github.com/kubernetes/website/issues/33479
kubeadm-6

```
29	9	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md
18	19	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_renew_etcd-peer.md
18	19	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_certs_renew_etcd-healthcheck-client.md
15	2	content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase.md
```

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md [hunk 1]
```diff
@@ -16,13 +16,19 @@ guide. You can file document formatting bugs against the
...
 -->
 
 ### 概要
 
 <!--
-Print a list of images kubeadm will use. The configuration file is used in case any images or image repositories are customized.
+Print a list of images kubeadm will use. The configuration file is used in case any images or image repositories are customized
 -->
 
 打印 kubeadm 要使用的镜像列表。配置文件用于自定义任何镜像或镜像存储库。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md:34]:
```suggestion
打印 kubeadm 要使用的镜像列表。配置文件用于自定义镜像或镜像存储库。
```

Comment (kinzhi) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_config_images_list.md]: Has been modified

**Pull request merged**